packages/interfaces/src/protocols.ts

```diff
@@ -36,10 +35,6 @@ export type NetworkConfig = StaticSharding | AutoSharding;
  * Options for using LightPush and Filter
  */
 export type ProtocolUseOptions = {
-  /**
```

Comment:
why these properties are removed?

Reply:
`autoRetry` should be kept and accounted in 
https://github.com/waku-org/js-waku/blob/1d68526e724155f76bb786239f475a774115ee97/packages/sdk/src/protocols/light_push/light_push.ts#L108

I haven't noticed it before, let's follow up but keep `autoRetry`

Reply:
`forceUseAllPeers` is in a way auto-retrying. The issue right now is that `options` are not being respected by the LightPush protocol, thus these options need to be re-thought. These are `PeerManagerOptions`